### ISSUE TITLE
Link UG names to either meetup or other info page from home page

### ISSUE THREAD

**Issue by iansltx** [MergePHP/mergephp-website#14]

**Comment by cookieguru** [comment]
https://github.com/MergePHP/mergephp-website/blob/4d88655ab9efe7df22525bf52347de5a5d721ccb/templates/index.twig.html#L10

**Comment by llbbl** [comment]
With #107 merged and #105 in progress, can this be closed? #107 added a `/groups/` page listing each affiliated user group with its meetup.com link, and the footer's "Participating Groups" label now links to it. #105 removes the per-meetup RSVP link mechanism this issue references.

**Comment by ericpoe** [comment]
Agreed. #107 seems to resolve this need. What do you think, @iansltx ?

**Comment by iansltx** [comment]
Yep.
